wp-content/themes/access/src/js/locations.js

```diff
@@ -1,54 +1,48 @@
 /* eslint-env browser */
-import jQuery from 'jquery';
-import Utility from 'modules/utility';
+// Core-js polyfills.
+// Core-js is made available as a dependency of @babel/preset-env
+import 'core-js/features/url-search-params';
```

Comment:
The office maps script is using URL Search Params to get params instead of the Utility method. This is a polyfill to support IE11.